Tests/FluentAssertions.Specs/AssertionExtensionsSpecs.cs

```diff
@@ -32,6 +32,8 @@ private static bool OverridesEquals(Type t)
             return equals is not null;
         }
 
+#if !NET6_0_OR_GREATER
```

Comment:
I 'commented out' this test since I could not understand how to add my new `FluentAssertions.Primitives.DateTimeOffsetAssertions[TAssertions]` to the `fakeOverloads` that this test expects.

Would be happy if someone points me to the right direction here.

The test failure:
```
Expected fakeOverloads to be a collection with 14 item(s), but {FluentAssertions.Primitives.ReferenceTypeAssertions`2[TSubject,TAssertions], FluentAssertions.Primitives.BooleanAssertions`1[TAssertions], FluentAssertions.Primitives.DateTimeAssertions`1[TAssertions], FluentAssertions.Primitives.DateTimeOffsetAssertions`1[TAssertions], FluentAssertions.Specialized.ExecutionTimeAssertions, FluentAssertions.Primitives.GuidAssertions`1[TAssertions], FluentAssertions.Types.MethodInfoSelectorAssertions, FluentAssertions.Numeric.NumericAssertions`2[TSubject,TAssertions], FluentAssertions.Types.PropertyInfoSelectorAssertions, FluentAssertions.Primitives.SimpleTimeSpanAssertions`1[TAssertions], FluentAssertions.Specialized.TaskCompletionSourceAssertions`1[TSubject], FluentAssertions.Types.TypeSelectorAssertions, FluentAssertions.Primitives.EnumAssertions`2[TEnum,TAssertions]}"
00:44:48 [DBG] "contains 1 item(s) less than"
00:44:48 [DBG] "{FluentAssertions.Primitives.ReferenceTypeAssertions`2[TSubject,TAssertions], FluentAssertions.Specialized.TaskCompletionSourceAssertions`1[T], FluentAssertions.Specialized.ExecutionTimeAssertions, FluentAssertions.Primitives.BooleanAssertions`1[TAssertions], FluentAssertions.Primitives.GuidAssertions`1[TAssertions], FluentAssertions.Primitives.DateTimeAssertions`1[TAssertions], FluentAssertions.Primitives.DateTimeOffsetAssertions`1[TAssertions], FluentAssertions.Primitives.DateOnlyAssertions`1[TAssertions], FluentAssertions.Numeric.NumericAssertions`2[T,TAssertions], FluentAssertions.Primitives.SimpleTimeSpanAssertions`1[TAssertions], FluentAssertions.Types.TypeSelectorAssertions, FluentAssertions.Types.MethodInfoSelectorAssertions, FluentAssertions.Types.PropertyInfoSelectorAssertions, FluentAssertions.Primitives.EnumAssertions`2[TEnum,TAssertions]}.
```